doc/Output JSON.json

```diff
@@ -240,18 +279,8 @@
           } ]
         },
         "entry": [ {
-          "reference": "Condition/8fa23827-fe32-409b-9fe6-b7c6a7fbf6c6"
+          "reference": "Condition/ed57c029-2795-45f5-9777-f4618e21f737"
         } ]
-      }, {
-        "title": "Past Surgical History",
-        "code": {
-          "coding": [ {
-            "system": "http://snomed.info/sct",
-            "code": "Coronary Artery Disease",
-            "display": "Past Surgical History Section"
-          } ],
-          "text": "Past Surgical History Section"
-        }
       }, {
         "title": "Mental Health Assesment",
```

Comment:
I thought we decided that we are not going to do the mental health assessment right now. Also section code and also related observation for pregnancy status is just wrong. that should not be part of mental assessment at all. For now, as decided, exclude the mental health assessment. If at all. this should be grouped under section "other observation" and if done under observation, it should be grouped under parent observation with SNOMED code "391281002"